app/minutes/controllers.py

```diff
@@ -44,9 +40,10 @@ def get_minute(user, user_data):
 
     membership = committee.members.filter_by(member = user).first()
 
-    if minute.private and (membership is None and not user.is_admin):
-        emit('get_minute', Response.PermError)
-        return
+    if minute.private:
+        if user is None or (membership is None and not user.is_admin):
```

Comment:
I wrote it like this because (I think) it's easier to parse.

Reply:
I like it better like that, personally.